flowcraft/generator/components/typing.py

```diff
@@ -161,4 +161,101 @@ def __init__(self, **kwargs):
         ]
 
 
+class Seqsero2Reads(Process):
+    """SeqSero2 for reads process template interface
 
+    This process is set with:
+
+        - ``input_type``: fastq
+        - ``output_type``: None
+        - ``ptype``: typing
+    """
+
+    def __init__(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        self.input_type = "fastq"
+        self.output_type = None
+
+        self.directives = {
+            "seqsero2_reads": {
+                "cpus": 1,
+                "memory": "{ 1.GB * task.cpus * task.attempt }",
+                "container": "ummidock/seqsero2",
+                "version": "alpha-test-1",
+                "cache": "false",
+                "scratch": "true"
+            }
+        }
+
+
+class Seqsero2Assembly(Process):
+    """SeqSero2 for assembly process template interface
+
+    This process is set with:
+
+        - ``input_type``: fasta
+        - ``output_type``: None
+        - ``ptype``: typing
+    """
+
+    def __init__(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        self.input_type = "fasta"
+        self.output_type = None
+
+        self.directives = {
+            "seqsero2_assembly": {
+                "cpus": 1,
+                "memory": "{ 1.GB * task.cpus * task.attempt }",
```

Comment:
Why is the memory relative to the number of cpus uses?

Reply:
Hi @tiagofilipe12 . I'm like a rock: I need a strong wind to make me move forward! Thanks for the review. Regarding the memory issue, I choose to scale the memory with the CPUs used because some software behave that way. I did some tests and in this particular case it seems to be independent of the CPUs used. I'll remove that part.